dom0/sd-gpg.sls

```diff
@@ -16,7 +16,7 @@ sd-gpg:
   qvm.vm:
     - name: sd-gpg
     - present:
-      - template: sd-workstation-template
```

Comment:
do you think it's worth circling back in https://github.com/freedomofpress/qubes-template-securedrop-workstation/ to update the name of the template? I  _think_ we would only need to change this line: https://github.com/freedomofpress/qubes-template-securedrop-workstation/blob/master/securedrop-workstation.conf#L26 (and rebuild, of course)

Reply:
No, I don't think it's worthwhile to alter the template name. You're right that it's inconsistent: `securedrop-workstation`, rather than `sd-workstation-template`. On the other hand, however, I see a useful distinction in that the naming convention here matches what's standard in the Qubes repos. To that end, we have a reason to leverage the slight inconsistency:

* `sd-<x>-template` for all cloned, Salt-managed VMs
* `securedrop-workstation` for the RPM-based image (currently only one)

Open to revising if folks find the naming consolidation useful, but would prefer to avoid the lengthy rebuilds at present, especially since we'd have to update the various config files in the builder repo, as well—having `securedrop-workstation.conf` emit `sd-workstation-template`, rather than `securedrop-workstation`, also has the potential to confuse.